statistics/handle/handletest/globalstats/globalstats_test.go

```diff
@@ -1127,6 +1128,8 @@ func TestGlobalIndexStatistics(t *testing.T) {
 	h.SetLease(time.Millisecond)
 	tk := testkit.NewTestKit(t, store)
 	tk.MustExec("use test")
+	sctx := tk.Session().(sessionctx.Context)
+	sctx.GetSessionVars().AnalyzePartitionMergeConcurrency = analyzePartitionMergeConcurrency
```

Comment:
This test doesn't seem to verify that the concurrency settings are set successfully?

Reply:
Yes, It is my problem.